Automattic/wp-calypso
# Assert the correct package manager is used for run commands

#### Changes proposed in this Pull Request

* Adds a guard to every script in package.json to assert that the right package manager (i.e. npm vs yarn) is being used.

This is part of yarn migration (see #40882). Once we have migrated in yarn and we are confident that everybody is used to the change, we can remove these guards.

#### Testing instructions

* Run your usual `npm run...` commands. Everything should work fine.
* Now run them with `yarn run...`. You should see the error "please use npm instead of yarn"
